### DISCUSSION TITLE
Feat: theme addition request

### DISCUSSION THREAD

**Discussion by eduwass** [aws/codewhisperer-command-line-discussions#208] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/q-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
> Please include a detailed description of the feature (and an image or screen recording, if applicable)

I created a theme for what previously was Fig, the code is here: https://github.com/withfig/themes/pull/34

Now that Fig has turned into [Amazon Q](https://aws.amazon.com/q/developer/) here's how you would activate this.

As you'll see there, it can be activated in Q like this:

1. Put `shades-of-purple.json` in: `/Applications/Amazon Q.app/Contents/Resources/themes`
2. Activate via CLI with:
`q theme shades-of-purple` and `q restart`

It would be cool if you guys add it.

Thanks!

**Comment by grant0417** [comment]
I have added your theme, it should be in the next release. Sorry for not watching that repo as much!

**Reply by eduwass** [reply]
Of course, no worries!
